### ISSUE TITLE
Update command error from missing flag attribute

### ISSUE THREAD

**Issue by guibom** [deadc0de6/catcli#39]
CATCLI has been working really well to index and list files.
However when I try to update a catalog, I get the following error: `AttributeError: 'AnyNode' object has no attribute 'flag'`

I'm not using a hash, and it looks like the entries do have the proper modified time.
Initially I thought it might be because I was indexing a mounted samba share, but the same thing happens when indexing/updating local files.

`$ catcli update Animation ~/mount/Animation/`

```
 +-+-+-+-+-+-+
 |c|a|t|c|l|i|
 +-+-+-+-+-+-+ v0.9.5

Traceback (most recent call last):
  File "/usr/local/bin/catcli", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/catcli/__init__.py", line 13, in main
    if catcli.catcli.main():
  File "/usr/local/lib/python3.10/dist-packages/catcli/catcli.py", line 342, in main
    cmd_update(args, noder, catalog, top)
  File "/usr/local/lib/python3.10/dist-packages/catcli/catcli.py", line 157, in cmd_update
    cnt = walker.reindex(path, root, top)
  File "/usr/local/lib/python3.10/dist-packages/catcli/walker.py", line 94, in reindex
    cnt = self._reindex(path, parent, top)
  File "/usr/local/lib/python3.10/dist-packages/catcli/walker.py", line 119, in _reindex
    node.flag()
AttributeError: 'AnyNode' object has no attribute 'flag'
```

**Comment by deadc0de6** [comment]
It's a bug! Sorry for that, I'm working on it and will release a fix with a new version ASAP

**Comment by deadc0de6** [comment]
Fixed in the [new version 0.9.6](https://pypi.org/project/catcli/)

**Comment by guibom** [comment]
Yep, it's working fine now! 
Thanks for the quick response and an awesome project.
